prboom2/src/sounds.c

```diff
@@ -247,4 +247,210 @@ sfxinfo_t S_sfx[] = {
   //e6y
   { "secret", false, 60, 0, -1, -1, 0 },
   { "gibdth", false, 60, 0, -1, -1, 0 },
+  // Everything from here up to 400 is reserved for future use.
```

Comment:
Or 500?

Reply:
Yeah, that's a silly oversight - it _was_ 400 until shortly before the commit, when Altazimuth requested another 100 reserved slots for EE. Either way, fixed.